### PR TITLE
Fix project explorer search

### PR DESCRIPTION
**Pull request by tobi321**
- Search bar in project explorer is sticky when scrolling
- Tabs in project explorer is sticky when scrolling
- If tab is selected it has a grey background

### CODE REVIEW

**Review by wehrlean — CHANGES_REQUESTED** [review]
Sadly it doesn't work properly.
- remove the bar if it is not needed
- the end of the bar is not on the right point -> see Screenshots on Trello
- on Firefox it doesn't work at all
- ~~search is not working, but I think that is another issue~~

**Comment by tobi321** [comment]
Tested on firefox and chrome

File: web/src/app/modules/navigation/modules/project-explorer/components/project-explorer.component.css [hunk 1]
```diff
@@ -1,4 +1,40 @@
...
+    height: 100%;
+    max-height: 100%;
+    width: 100%;
+    display: inline-flex;
+    flex: 1 1 0%;
+    flex-direction: column;
+}
+
+#list-wrapper {
+    overflow: scroll;
```
Comment (wehrlean) [inline web/src/app/modules/navigation/modules/project-explorer/components/project-explorer.component.css]: Change overflow to 'auto'
Add 'height: 100%'

File: web/src/app/modules/navigation/modules/project-explorer/components/project-explorer.component.html [hunk 2]
```diff
@@ -1,48 +1,62 @@
...
+            <tr *ngFor="let element of searchResults; let i = index;">
+                <td>
+                    <a [navigationTarget]="element">
+                        <icon-selector [model]="element" expanded="false"></icon-selector>{{element['extId'] ?
+                        element['extId'] + ': ' : ''}}{{element.name}}
+                    </a>
+                </td>
+            </tr>
+        </tbody>
+    </table>
```
Comment (wehrlean) [inline web/src/app/modules/navigation/modules/project-explorer/components/project-explorer.component.html]: The search-result is not scrollable

**Review by wehrlean — CHANGES_REQUESTED** [review]
It works, nice :) 
Only two small additions, see comments.

**Comment by tobi321** [comment]
Search Results are now also scrollable with sticky 'Results' header.

**Review by wehrlean — APPROVED** [review]
Yeah, works fine!!!!

**Pull request merged**